### PR TITLE
OCM-9706 | ci: Allow COMPUTE_MACHINE_TYPE as env

### PR DESCRIPTION
**Pull request by radtriste**
**What this PR does / why we need it**:
Create CI profile for ARM multiarch checks

**Which issue(s) this PR fixes** 
https://issues.redhat.com/browse/OCM-9706

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

**Comment by radtriste** [comment]
```
Will run 1 of 118 specs
------------------------------
[BeforeSuite] 
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/e2e_suite_test.go:24
[BeforeSuite] PASSED [0.000 seconds]
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
Verify cluster multiarch is set correctly - [id:75107] [day1-post, High]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verification_post_day1_test.go:390
• [0.000 seconds]
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 118 Specs in 0.001 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 117 Skipped
PASS
```

File: tests/ci/profiles/tf_hcp_cluster_profiles.yml [hunk 1]
```diff
@@ -208,4 +208,32 @@ profiles:
     oidc_config: "managed" # id:72493
     admin_enabled: false # Not yet supported
     unified_acc_role_path: ""
-    ingress_listening_method: "internal" # id:72516
\ No newline at end of file
+    ingress_listening_method: "internal" # id:72516
+
+- as: rosa-hcp-arm # id:75107
```
Comment (yasun1) [inline tests/ci/profiles/tf_hcp_cluster_profiles.yml:213]: I have the same thinking in https://github.com/openshift/rosa/pull/2268 about whether we could use the existing profile and a new prow CI job to achieve the testing aim.

Comment (radtriste) [reply tests/ci/profiles/tf_hcp_cluster_profiles.yml]: Removed the profile. Added the COMPUTE_MACHINE_TYPE env configuration

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**